include/tvm/relay/attrs/nn.h

```diff
@@ -937,7 +937,15 @@ struct DenseAttrs : public tvm::AttrsNode<DenseAttrs> {
 
 /*! \brief Attributes for sparse_dense operator */
 struct SparseDenseAttrs : public tvm::AttrsNode<SparseDenseAttrs> {
-  TVM_DECLARE_ATTRS(SparseDenseAttrs, "relay.attrs.SparseDenseAttrs") {}
+  bool sparse_lhs;
```

Comment:
need we naming it with suffix `lhs`?

Reply:
Thanks @FrozenGene! Lhs is used to indicate which matrix is sparse in the Op. You are most welcome to suggest a better name!